streams/src/main/java/org/apache/kafka/streams/processor/internals/namedtopology/KafkaStreamsNamedTopologyWrapper.java

```diff
@@ -149,6 +149,10 @@ public NamedTopologyBuilder newNamedTopologyBuilder(final String topologyName) {
         return Optional.ofNullable(topologyMetadata.lookupBuilderForNamedTopology(name)).map(InternalTopologyBuilder::namedTopology);
     }
 
+    public Collection<NamedTopology> getAllTopologies() {
+        return getAllTopologies();
```

Comment:
this looks like an infinite loop?

Reply:
oops. I though I was calling that on topology metadata. Fixed!